db/blob/blob_file_reader.cc

```diff
@@ -20,11 +20,13 @@
 
 namespace ROCKSDB_NAMESPACE {
 
-Status BlobFileReader::Create(
-    const ImmutableCFOptions& immutable_cf_options,
-    const FileOptions& file_options, uint32_t column_family_id,
-    HistogramImpl* blob_file_read_hist, uint64_t blob_file_number,
-    std::unique_ptr<BlobFileReader>* blob_file_reader) {
+Status BlobFileReader::Create(const ImmutableCFOptions& immutable_cf_options,
+                              const FileOptions& file_options,
+                              uint32_t column_family_id,
+                              HistogramImpl* blob_file_read_hist,
+                              uint64_t blob_file_number,
+                              std::unique_ptr<BlobFileReader>* blob_file_reader,
+                              const std::shared_ptr<IOTracer>& io_tracer) {
```

Comment:
Similarly to the above, `blob_file_reader` is the output parameter of this method; could we keep it in the last position?